x/arkeo/types/events.go

```diff
@@ -20,13 +20,15 @@ func NewOpenContractEvent(openCost int64, contract *Contract) EventOpenContract
 		Service:            contract.Service.String(),
 		Client:             contract.Client,
 		Delegate:           contract.Delegate,
-		Type:               contract.Type,
+		UserType:           contract.UserType,
+		MeterType:          contract.MeterType,
 		Height:             contract.Height,
 		Duration:           contract.Duration,
 		Rate:               contract.Rate,
 		OpenCost:           openCost,
 		Deposit:            contract.Deposit,
 		SettlementDuration: contract.SettlementDuration,
+		Restrictions:       contract.Restrictions,
```

Comment:
do we need restrictions in events?

Reply:
I am not sure why it would be treated differently than any other of these fields.  If sentinel wants to reconstruct the contract info from the event it is needed.  Alternatively, we could remove almost all of these fields from the event and force consumers to instead query against the chain for the contract details.

Reply:
okay, nevermind.